yoyodyne/train.py

```diff
@@ -323,6 +323,10 @@ def main() -> None:
     trainer = get_trainer_from_argparse_args(args)
     datamodule = get_datamodule_from_argparse_args(args)
     model = get_model_from_argparse_args(args, datamodule)
+    # Logs number of model parameters.
+    if args.log_wandb:
+        n_params = sum(p.numel() for p in model.parameters())
+        wandb.config.update({"n_model_params": n_params})
```

Comment:
Correct me if I'm wrong but didn't we establish that `config` is just a dictionary, or at least has a `__getitem__` like one? Then you could rewrite this:

```python
wandb.config["n_model_params"] = sum(p.numel() for p in model.parameters())
```

or whatever.